admin/views/licenses.php

```diff
@@ -148,36 +148,37 @@
 			<?php
 			$extension = $extensions->get( 'wordpress-seo-premium' );
 			$extensions->remove( 'wordpress-seo-premium' );
-			?>
-			<h2>
-				<?php
-				printf(
-					/* translators: %1$s expands to Yoast SEO Premium */
-					esc_html__( '%1$s, take your optimization to the next level!', 'wordpress-seo' ),
-					'<span class="yoast-heading-highlight">' . $extension->get_title() . '</span>'
-				);
+			if ( ! $extensions->is_activated( 'wordpress-seo-premium' ) ) :
 				?>
-			</h2>
-
-			<ul class="yoast-seo-premium-benefits yoast-list--usp">
-				<li class="yoast-seo-premium-benefits__item">
-					<span class="yoast-seo-premium-benefits__title"><?php esc_html_e( 'Redirect manager', 'wordpress-seo' ); ?></span>
-					<span class="yoast-seo-premium-benefits__description"><?php esc_html_e( 'create and manage redirects from within your WordPress install.', 'wordpress-seo' ); ?></span>
-				</li>
-				<li class="yoast-seo-premium-benefits__item">
-					<span class="yoast-seo-premium-benefits__title"><?php esc_html_e( 'Multiple focus keywords', 'wordpress-seo' ); ?></span>
-					<span class="yoast-seo-premium-benefits__description"><?php esc_html_e( 'optimize a single post for up to 5 keywords.', 'wordpress-seo' ); ?></span>
-				</li>
-				<li class="yoast-seo-premium-benefits__item">
-					<span class="yoast-seo-premium-benefits__title"><?php esc_html_e( 'Social previews', 'wordpress-seo' ); ?></span>
-					<span class="yoast-seo-premium-benefits__description"><?php esc_html_e( 'check what your Facebook or Twitter post will look like.', 'wordpress-seo' ); ?></span>
-				</li>
-				<li class="yoast-seo-premium-benefits__item">
-					<span class="yoast-seo-premium-benefits__title"><?php esc_html_e( 'Premium support', 'wordpress-seo' ); ?></span>
-					<span class="yoast-seo-premium-benefits__description"><?php esc_html_e( 'gain access to our 24/7 support team.', 'wordpress-seo' ); ?></span>
-				</li>
-			</ul>
-
+				<h2>
+					<?php
+					printf(
+					/* translators: %1$s expands to Yoast SEO Premium */
+						esc_html__( '%1$s, take your optimization to the next level!', 'wordpress-seo' ),
```

Comment:
This line should not be in the if statement, as it makes the title "Yoast SEO Premium, take your optimization to the next level!" invisible when Premium is activated. Or perhaps keep it in the if statement but add an alternative title when activated/installed. 

<img width="1501" alt="screenshot 2018-03-01 17 15 58" src="https://user-images.githubusercontent.com/16388834/36855906-0014d436-1d75-11e8-884f-54e6f6db3c43.png">